Cargo.toml

```diff
@@ -15,5 +15,12 @@ chrono = "0.4.38"
 hex = "0.4.3"
 clap = { version = "4.5.7", features = ["derive"] }
 
+[features]
+
+# Enables feature `min_specialization` of nightly compiler and provides
```

Comment:
I which production project can we use unstable rust?

Reply:
It's entirely up to consumers of the lib whether they want to enable this feature or not. By default it's disabled.

The purpose of this feature is only to provide optimalization for converting a collection of `RpcValue`s into `RpcValue`, where a user code decides to handle all the conversions in the uniform way - calling `.into()` without any customization for specific data.

Reply:
We have only one consumer currently - us. So this code is effectively dead. We can keep it until `min_specialization` will be stabilized, hopefully there will be some external consumers this time. My concern was just to know, how can be this feature utilized now.

Reply:
Whether or not there are currently any external consumers  shouldn't be the concern _here_. Anybody can start a project at any time utilizing the feature if they consider it meaningful.

The `min_specialization` has been around for a while and it's highly demanded. I'd not think it will get removed. However, it might not get stabilized for another couple of years, which would mean we should either give up on this feature completely, or at least leave it there as an option, disabled by default, which is what I propose.

Reply:
Well, we can keep it and revisit it when `specialization` will be stabilized even if it is far from finished https://github.com/rust-lang/rust/issues/31844.

Reply:
I still wonder, why specialization generates necessity of macro used on lines `rpcvalue.rs:208` - `rpcvalue.rs:222`, which wasn't needed before. My naive understanding is, that just special cases should be introduced, but the generic `impl` should remain the same.

Reply:
The `cfg` macros inside the `impl`s are necessary because of the `default` keyword/marker in the definition of the generic implementations of `from` methods, which can appear only if the compiler feature is enabled.